### ISSUE TITLE
Feature Request: Low Latency / Game  Mode

### ISSUE THREAD

**Issue by JackByrne** [TheElixZammuto/moonlight-xbox#112]
Xbox supports a setting named "Allow auto low-latency mode". If your TV supports the corresponding feature, e.g. "Game Mode" then it will be auto-set when a game starts. 

The setting is found under General > TV & display options > Advanced > Video modes. 

Would it be possible to add a setting into the moonlight application to run "as a game" so this would be supported? This may allow for improved input lag etc.

If you need more information then let me know.

Thanks.

**Comment by harun911** [comment]
I don't think this is possible at all, due to the nature of moonlight being a video stream and not a game running natively. This means that whenever a game drops frames, the host will always be outputting the preset among of frames. So for instance, it you set 60fps, the host will still capture 60fps even though the game drops below or goes higher then that. The key for the a smooth stream, is to find a way to get your game to deliver a smooth constant fps by tweaking the settings.

**Comment by ReenigneArcher** [comment]
> I don't think this is possible at all, due to the nature of moonlight being a video stream and not a game running natively.

I think you may be misunderstanding the request. It should be possible, as it would just require changing the classification from an "app" to a "game". But I don't know if there are extra hoops to jump through with MSFT, or the pros and cons of an "app" versus a "game".
